libc/test/src/math/CMakeLists.txt

```
@@ -2430,9 +2430,72 @@ add_fp_unittest(
   HDRS
     SubTest.h
   DEPENDS
+    libc.src.errno.errno
+    libc.hdr.fenv_macros
     libc.src.math.dsubl
 )
```

libc/test/src/math/SubTest.h doesn't use these.

```suggestion
  DEPENDS
    libc.src.math.dsubl
)
```